packages/core/src/js/sdk.tsx

```diff
@@ -153,18 +153,18 @@ export function init(passedOptions: ReactNativeOptions): void {
  */
 export function wrap<P extends Record<string, unknown>>(
   RootComponent: React.ComponentType<P>,
-  options?: ReactNativeWrapperOptions
+  options?:  ReactNativeWrapperOptions
 ): React.ComponentType<P> {
   const profilerProps = {
-    ...(options?.profilerProps ?? {}),
+    ...(options?.profilerProps ?? { removeUpdateProps: true }),
     name: RootComponent.displayName ?? 'Root',
-    updateProps: {}
+    updateProps: options?.profilerProps
```

Comment:
I thought we would keep the `updateProps: {}`?

Reply:
Sorry, probably a misunderstanding, fixed here https://github.com/getsentry/sentry-react-native/pull/4740/commits/517161cb2485951ec396690a41df37d412b0e3cd